autonomx/main.cpp

```diff
@@ -59,15 +59,15 @@ int main(int argc, char *argv[]) {
     const char* applicationName = "autonomX";
     const char* applicationVersion = "0.1.1";
     const char* organizationName = "Xmodal";
-    const char* extensionName = "atnx";
```

Comment:
I don't understand why you would want to remove that constants. Constants make the code easier to read. I know this constant is used in only one place, but it does't make our application faster to hard-code it inline instead of using a constant. I think it makes the code less readable to remove that constant.